src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -3587,25 +3569,22 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         }
 
         const auto panel = SwapChainPanel();
-        const auto scale = panel.CompositionScaleX();
         const auto offset = panel.ActualOffset();
 
         // Update the tooltip with the URI
         HoveredUri().Text(uriText);
 
         // Set the border thickness so it covers the entire cell
         const auto charSizeInPixels = CharacterDimensions();
-        const auto htInDips = charSizeInPixels.Height / scale;
-        const auto wtInDips = charSizeInPixels.Width / scale;
-        const Thickness newThickness{ wtInDips, htInDips, 0, 0 };
+        const Thickness newThickness{ charSizeInPixels.Height, charSizeInPixels.Width, 0, 0 };

```

Comment:
Wait up - CharacterDimensions calls FontSizeInDips **and should not be in pixels**

Reply:
I'm always a little confused if XAML wants DIPs or pixels, but I think it wants DIPs in this case, right? Assuming that is correct, I've just changed the variable name and left the rest as-is.